node/send_msg_using_node/index.js

```diff
@@ -0,0 +1,26 @@
+// Include voyage module
```

Comment:
would you please put everything in `node/send_message` instead of `node/send_msg_using_node`?